src/builtins/status.rs

```diff
@@ -475,7 +475,7 @@ pub fn status(parser: &Parser, streams: &mut IoStreams, args: &mut [&wstr]) -> B
             #[cfg(not(feature = "embed-data"))]
             {
                 streams.err.append(wgettext_fmt!(
```

Comment:
This should be appendln instead of adding the newline - which has the advantage of not leaving the newline in the translated string.

Reply:
Sounds good, will fix in a moment. FYI, this is not done consistently currently, see e.g.

https://github.com/fish-shell/fish-shell/blob/471fa78bdc383cae6c5c838c2c763a337ef850e1/src/builtins/status.rs#L242

https://github.com/fish-shell/fish-shell/blob/471fa78bdc383cae6c5c838c2c763a337ef850e1/src/builtins/status.rs#L233

I don't want to fix it myself, since I don't know whether those strings were already translated or not.

Reply:
No problem, and makes sense. Fixed.